src/commands/compress.rs

```diff
@@ -55,6 +55,10 @@ pub fn compress_files(
                 encoder,
                 level.map_or_else(Default::default, |l| bzip2::Compression::new((l as u32).clamp(1, 9))),
             )),
+            Bzip3 => Box::new(
+                // Unwrap is safe when using a valid block size
+                bzip3::write::Bz3Encoder::new(encoder, bytesize::ByteSize::mib(5).0 as usize).unwrap(),
```

Comment:
I think we should default to 16MiB here, which is consistent with bzip3's cli

also I'm not sure about using `bytesize` here

```suggestion
                bzip3::write::Bz3Encoder::new(encoder, 16 * 2.pow(20)).unwrap(),
```

Reply:
done in [2289def](https://github.com/ouch-org/ouch/pull/522/commits/2289def77b29b6da11e277763826c82bbd41f7d3)